src/modules/person/api/person.controller.ts

```diff
@@ -52,28 +52,74 @@ import { PersonendatensatzResponse } from './personendatensatz.response.js';
 import { PersonScope } from '../persistence/person.scope.js';
 import { ScopeOrder } from '../../../shared/persistence/index.js';
 import { PersonFactory } from '../domain/person.factory.js';
+import { PersonPermissions } from '../../authentication/domain/person-permissions.js';
+import { Permissions } from '../../authentication/api/permissions.decorator.js';
+import { OrganisationID } from '../../../shared/types/index.js';
+import { RollenSystemRecht } from '../../rolle/domain/rolle.enums.js';
+import { DataConfig, ServerConfig } from '../../../shared/config/index.js';
+import { ConfigService } from '@nestjs/config';
+import { PersonUc } from '../domain/person.uc.js';
 
 @UseFilters(SchulConnexValidationErrorFilter)
 @ApiTags('personen')
 @ApiBearerAuth()
 @ApiOAuth2(['openid'])
 @Controller({ path: 'personen' })
 export class PersonController {
+    public readonly ROOT_ORGANISATION_ID: string;
+
     public constructor(
         private readonly personenkontextUc: PersonenkontextUc,
         private readonly personRepository: PersonRepository,
         private readonly personFactory: PersonFactory,
+        private readonly personUc: PersonUc,
         @Inject(getMapperToken()) private readonly mapper: Mapper,
-    ) {}
+        config: ConfigService<ServerConfig>,
+    ) {
+        this.ROOT_ORGANISATION_ID = config.getOrThrow<DataConfig>('DATA').ROOT_ORGANISATION_ID;
+    }
+
+    //check that logged-in user is allowed to update person
+    private async getPersonOrThrowException(
+        params: PersonByIdParams,
```

Comment:
Any reason why this accepts the entire params-object instead of just a `PersonID`?